alpharaw/mzml.py

```diff
@@ -10,10 +10,38 @@
 
 
 class MzMLReader(MSData_Base):
+    """
+    Load mzml file as `:class:`MSData_Base` structure.
+    Register "mzml" in :data:`ms_reader_provider`.
+
+    Parameters
+    ----------
+    filename : str
```

Comment:
a parameter `filename` containing a file path? should we make `filename` deprecated and add a parameter "file_path"?
also:  absolute? relative? :-)